content/en/security/cloud_security_management/guide/eBPF-free-agent.md

```diff
@@ -0,0 +1,421 @@
+
+You also have to connect the container to Datadog on port 5678 by doing one of the following:
+
+- Launch both containers with the `--network` host option.
+- Use the [Docker network][6] feature to run both containers on the same bridge network.
+
+{{% /tab %}}
+
+{{% tab "Attach mode" %}}
+The wrap mode is recommended because the attach mode has the following limitations: 
+
+- It misses all initialization made by the application until Datadog attaches to it.
+- - When attaching, Datadog cannot set up a seccomp profile.
+- More performance overhead.
+- If the traced application restarts, Datadog must ensure that the tracer restarts also.
+
+The attach mode differs from the wrap mode by attaching directly the tracer on an already running application, like this:
+
+```shell
+sudo /opt/datadog-agent/embedded/bin/cws-instrumentation trace --pid 2301
+```
+
+Several PIDs can be attached at once:
+
+```shell
+sudo /opt/datadog-agent/embedded/bin/cws-instrumentation trace --pid 2301 --pid 2302 --pid 2303
+```
+
+The following examples show how the tracer can be integrated within applications for different deployment types.
+
+#### Linux systemd service
+
+If you already have an init script, here is an example of how to integrate the wrapper using a new systemd service:
+
+```shell
+[Unit]
+Description=Datadog CWS instrumentation attach to my application
+After=datadog-agent-sysprobe.service my-app.service
+
+[Service]
+ExecStart=/bin/bash -c "/opt/datadog-agent/embedded/bin/cws-instrumentation trace $(for pid in $(pidof myapp); do echo --pid $pid; done)"
+Restart=on-failure
+
+[Install]
+WantedBy=multi-user.target
+```
+
+#### Linux sysvinit service
+
+If you already have an init script, here is an example of how to integrate the tracer using a new sysvinit service:
+
+```shell
+#!/bin/sh
+set -e
+### BEGIN INIT INFO
+# Provides:           dd_tracing_my_app
+# Required-Start:     $network
+# Required-Stop:      $network
+# Default-Start:      2 3 4 5
+# Default-Stop:       0 1 6
+# Short-Description:  Datadog tracing of my application
+# Description: Datadog tracing of my application
+### END INIT INFO
+
+# Start the service
+start() {
+        echo "Starting tracing my app"
+        /opt/datadog-agent/embedded/bin/cws-instrumentation trace $(for pid in $(pidof myapp); do echo --pid $pid; done) &
+}
+
+
+# Stop the service
+stop() {
+       echo "Stopping my app"
+	pkill -f /opt/datadog-agent/embedded/bin/cws-instrumentation
+}
+
+### main logic ###
+case "$1" in
+  start)
+        start
+        ;;
+  stop)
+        stop
+        ;;
+  restart)
+        stop
+        start
+        ;;
+  *)
+        echo $"Usage: $0 {start|stop|restart}"
+        exit 1
+esac
+
+exit 0
+```
+
+#### Docker
+
+To attach the wrapper to a Docker image running an application, use the following Dockerfile:
+
+```shell
+FROM gcr.io/datadoghq/agent:latest
```

Comment:
It should be the same, but to stay consistent with the other parts I would replace the `latest` tag by `7`